web/lib/ozoneframework/php/core/Database/BaseDBPeer.php

```diff
@@ -223,6 +222,11 @@ protected function insert($object){
 					}
 				}
 
+				// Process the JSONB data type.
+				if($fieldType === 'jsonb'){
```

Comment:
This should also support the `JSON` type:
```suggestion
				if($fieldType === 'json' && $fieldType === 'jsonb'){
```